src/accelerate/utils/modeling.py

```diff
@@ -1871,3 +1871,37 @@ def get_mixed_precision_context_manager(native_amp: bool = False, autocast_kwarg
             return torch.autocast(device_type=device_type, **autocast_kwargs)
     else:
         return contextlib.nullcontext()
+
+
+def get_grad_scaler(use_fsdp: bool = False, **kwargs):
```

Comment:
Instead of `use_fsdp`, how about passing the `DistributedType` as argument? That gives more room for flexibility in the future and wouldn't make the function API more complicated.